main.py

```diff
@@ -2,6 +2,44 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    def fill_in_tables() -> None:
+        genres = ["Western", "Action", "Dramma"]
+
+        for genre in genres:
+            Genre.objects.create(name=genre)
+
+        actors = [
+            ("George", "Klooney"),
+            ("Kianu", "Reaves"),
+            ("Scarlett", "Keegan"),
+            ("Will", "Smith"),
+            ("Jaden", "Smith"),
+            ("Scarlett", "Johansson")
+        ]
+
+        for actor in actors:
+            Actor.objects.create(first_name=actor[0], last_name=actor[1])
+
+    fill_in_tables()
+
+    def update_tables() -> None:
+        Genre.objects.filter(name="Dramma").update(name="Drama")
+        Actor.objects.filter(last_name="Klooney").update(last_name="Clooney")
```

Comment:
I decided to create nested functions to divide logical parts of main function. For me different actions in one function look less readable, thats why I wrote in nested way.

Reply:
Your main function still performs multiple actions. All other functions are executed inside main(). I wonder if there were some reasons that made you to choose nested functions instead of few separete ones

Reply:
After your arguments I began to doubt in my chose. I totally agree with you that the best solution is to use a couple of separate functions, but the task says to create 'one main fuc'.